react/training-project/src/app/components/Board/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+
+import Square from '../Square';
+
+import style from './styles.scss';
+
+class Board extends Component {
+  renderSquare = () => <Square />;
+  render() {
+    const status = 'Next player: X';
+    return (
+      <React.Fragment>
+        <div className={style.status}>{status}</div>
```

Comment:
Don't use divs to contain text. Use text tags for that, like h1-h6, p, spans, etc.